<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Создать уведомление с привязкой к времени</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDateEdit" name="timeDateInput">
     <property name="dateTime">
      <datetime>
       <hour>21</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2025</year>
       <month>5</month>
       <day>21</day>
      </datetime>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTimeEdit" name="timeTimeInput">
     <property name="dateTime">
      <datetime>
       <hour>6</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="timeTitleInput">
     <property name="placeholderText">
      <string>Заголовок</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="timeDescriptionInput">
     <property name="placeholderText">
      <string>Содержание</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="timeAddButton">
     <property name="text">
      <string>Добавить уведомление</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Создать уведомление с задержкой в минутах</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QSpinBox" name="waitTimeInput"/>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="waitTitleInput">
     <property name="placeholderText">
      <string>Заголовок</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="waitDescriptionInput">
     <property name="placeholderText">
      <string>Содержание</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="waitAddButton">
     <property name="text">
      <string>Добавить уведомление</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="notificationsListButton">
     <property name="text">
      <string>Просмотр списка уведомлений</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>Включить уведомления</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
